src/game/interactables_2/sky_canyon/whirlwind.c

```diff
@@ -21,12 +21,15 @@ typedef struct {
     /* 0x0E */ s16 top;
     /* 0x10 */ s16 right;
     /* 0x12 */ s16 bottom;
-    /* 0x14 */ s32 someX;
-    /* 0x18 */ s32 someY;
+    /* 0x14 */ s32 centerX;
+    /* 0x18 */ s32 centerY;
     /* 0x1C */ u16 height;
     /* 0x1E */ u16 width;
-    /* 0x20 */ u16 unk20;
-    /* 0x22 */ u16 unk22;
+    /* 0x20 */ u16 quarterWidth;
+
+    // unused
+    /* 0x22 */ u8 padding22[2];
```

Comment:
You should be able to delete this. The pointer after this will align to 4

Reply:
Yeah already tried that before, without issue. 
Just wasn't sure whether we want to keep padding for documentation purposes or not! :D